server/core/runtime/plan_step_runner.go

```diff
@@ -63,14 +63,14 @@ func (p *PlanStepRunner) isRemoteOpsErr(output string, err error) bool {
 	if err == nil {
 		return false
 	}
-	return strings.Contains(output, remoteOpsErr01114) || strings.Contains(output, remoteOpsErr012) || strings.Contains(output, remoteOpsErr100)
+	return strings.Contains(output, remoteOpsErr110) || strings.Contains(output, remoteOpsErr01114) || strings.Contains(output, remoteOpsErr012) || strings.Contains(output, remoteOpsErr100)
 }
 
 // remotePlan runs a terraform plan command compatible with TFE remote
 // operations.
 func (p *PlanStepRunner) remotePlan(ctx command.ProjectContext, extraArgs []string, path string, tfVersion *version.Version, planFile string, envs map[string]string) (string, error) {
 	argList := [][]string{
-		{"plan", "-input=false", "-refresh"},
+		{"plan", "-input=false", "-refresh", "-no-color"},
```

Comment:
Before the change, the output in the comment contains color code and doesn't look very nice.
<img width="912" alt="Screen Shot 2022-12-13 at 2 24 11 PM" src="https://user-images.githubusercontent.com/17621872/207458551-0b18b318-0f7a-4742-a9c5-be38b82faf15.png">

Also, I was debating whether to check `tfVersion` before adding the extra `-no-color` argument, and felt it might be fine to ignore it for better readability in the main code and the test cases. `-no-color` seems to have been introduced in Terraform since very early on (at least before v0.11).